src/iuse_actor.h

```diff
@@ -15,6 +15,7 @@
 #include "explosion.h"
 #include "game_constants.h"
 #include "iuse.h"
+#include "npc.h"
```

Comment:
```suggestion
class npc_template;
```

There is really no need to drag in the full "npc.h" header.